Mathlib/LinearAlgebra/PiTensorProduct.lean

```diff
@@ -613,6 +733,21 @@ theorem reindex_refl : reindex R s (Equiv.refl ι) = LinearEquiv.refl R _ := by
   congr
 #align pi_tensor_product.reindex_refl PiTensorProduct.reindex_refl
 
+variable {t : ι → Type*}
+variable [∀ i, AddCommMonoid (t i)] [∀ i, Module R (t i)]
+
+/-- Re-indexing the components of the tensor product by an equivalence `e` is compatible
+with `PiTensorProduct.map`.-/
+theorem map_reindex (f : Π i, s i →ₗ[R] t i) (e : ι ≃ ι₂) :
```

Comment:
To match `TensorProduct`, can you rename this to `map_comp_reindex`, and remove the `apply` from the other lemma?

Reply:
To match `TensorProduct`, it should even be `map_comp_reindex_eq`, I believe.